### PR TITLE
Add and configure scss

### PR DESCRIPTION
**Pull request by m-henderson**

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -6,6 +6,12 @@ module.exports = {
   plugins: [
     'gatsby-plugin-react-helmet',
     {
+      resolve: `gatsby-plugin-sass`,
```
Comment (systimotic) [inline gatsby-config.js]: The backticks should be single quotes to make the linter happy.

File: gatsby-config.js [hunk 2]
```diff
@@ -6,6 +6,12 @@ module.exports = {
...
+      options: {
+        precision: 8,
```
Comment (systimotic) [inline gatsby-config.js]: The comma on this line should be removed to make the linter even more happy.

**Comment by m-henderson** [comment]
@systimotic I push some more commits, have a look now.

**Comment by m-henderson** [comment]
@systimotic lol, that would be cool! Breast cancer awareness month is during October here in America!

**Pull request merged**